### PR TITLE
Changed "yank setup" to "yank prepare" throughout to avoid weird nose bug

### PR DESCRIPTION
**Pull request by jchodera**
This attempts to address #187. Lots of changes were necessary---not sure if I got everything changed.

### CODE REVIEW

File: examples/SAMPL-implicit/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The mbondi2 radii are used, with OBC GBSA in YANK.
 
 Set up the simulation to alchemically decouple benzene, putting all the output files in `output/`:
 ```tcsh
-yank setup binding amber --setupdir=setup --ligname=MOL --store=output --iterations=1000 --restraints=harmonic --gbsa=OBC2 --temperature=300*kelvin --minimize --verbose
+yank preparebinding amber --setupdir=setup --ligname=MOL --store=output --iterations=1000 --restraints=harmonic --gbsa=OBC2 --temperature=300*kelvin --minimize --verbose
```
Comment (kyleabeauchamp) [inline examples/SAMPL-implicit/README.md]: space

Comment (pgrinaway) [inline examples/SAMPL-implicit/README.md]: prepare binding ?

**Comment by pgrinaway** [comment]
I think [this error](https://travis-ci.org/choderalab/yank/builds/47839560#L1348-L1369) refers to retaining the old `yank setup binding` instead of the new `yank prepare binding`

**Comment by jchodera** [comment]
Is there a way to get nose to print out what file it's processing tests for?  Trying to figure out where these errors come from is becoming infuriating.

**Comment by kyleabeauchamp** [comment]
By default I think nose prints `filename.functionname()`, but that gets replaced by the docstring when present.

**Comment by bas-rustenburg** [comment]
Perhaps

```
  --logging-format=FORMAT
                        Specify custom format to print statements. Uses the
                        same format as used by standard logging handlers.
                        [NOSE_LOGFORMAT]
```

Could be useful?

**Comment by jchodera** [comment]
There's also some plugins that may help:
https://pypi.python.org/pypi/nose-progressive/

**Comment by jchodera** [comment]
Looks like we're just failing on the push to AWS because I haven't finished configuring the AWS encrypted keys.  If this looks good to everyone else, we should probably merge it.

**Pull request merged**